File: AD-HOC/2295.js
var lines = input.split(' ').map(Number);

let litA = lines[0], litG = lines[1];
let rendA = lines[2], rendG = lines[3];

if (rendA/litA > rendG/litG) {
  console.log('A');
} else if (rendA/litA < rendG/litG) {
  console.log('G');
} else {
  console.log('G');
}

File: AD-HOC/2295.py
entrada = list(map(float, input().split()))

litA = entrada[0]
litG = entrada[1]
rendA = entrada[2]
rendG = entrada[3]

if (rendA/litA > rendG/litG):
  print('A')
elif (rendA/litA < rendG/litG):
  print('G')
else:
  print('G')
